### PR TITLE
logging interesting events

### PR DESCRIPTION
**Pull request by adon-at-work**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js [hunk 1]
```diff
@@ -397,6 +401,16 @@ YmailApi.StormUI.prototype.displayFailure = function(message, opt_type) {
 };
 
 
+/**
+ * Log an interesting event
+ * @param {string} name
+ * @param {*} tags
```
Comment (mgoenka) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: You may want to specify that you can only have a maximum of 5 different tags.

Comment (adon-at-work) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: ok. will add a comment. do we have any more further restrictions on what can be put in tags? say, can we use array of string, i.e., calling it like `log('yme_compose', {recipients: ['a', 'b', 'c']})`

Comment (mgoenka) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: The tags cannot have indeterministic values, i.e. the possibilities should be finite.

File: src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js [hunk 2]
```diff
@@ -319,6 +321,8 @@ YmailApi.StormUI.prototype.dispatchOpenMessage = function(node, data) {
         body: text,
         quotedBody: quotedText
       }, bubbles: true}));
+
+    // this.log('yme_read', {});
```
Comment (mgoenka) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: The empty object for `tags` is optional.

File: src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js [hunk 3]
```diff
@@ -582,6 +600,12 @@ YmailApi.StormUI.DraftApi.prototype.send = function(draft) {
   var result = this.sendResult_(); // rely on the next send event. no auto send
   this.set(draft);
   this.composeView_.handleComposeAction('send');
+
+  // this.log('yme_send', {
+  //   to: draft.to,
```
Comment (mgoenka) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: `to` can have infinite variations, so cannot be logged.

File: src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js [hunk 4]
```diff
@@ -582,6 +600,12 @@ YmailApi.StormUI.DraftApi.prototype.send = function(draft) {
...
+  //   cc: draft.cc,
```
Comment (mgoenka) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: `cc` can have infinite variations, so cannot be logged.

File: src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js (hunk 1 above)
Comment (adon-at-work) [inline src/javascript/crypto/e2e/extension/helper/stubs/ymail.storm.js]: so, what can we do if we want to log version number?

**Pull request merged**